src/NuGet.Indexing/IndexingUtils.cs

```diff
@@ -28,7 +29,7 @@ public static class IndexingUtils
                 {
                     throw;
                 }
-                Trace.TraceInformation("Unable to load {0}. Exception Message : {1}", name, e.Message);
+                logger.LogInformation("Unable to load {0}. Exception Message : {1}", name, e.Message);
```

Comment:
No one gave me a reason why this was logged as information, so I updated to log it as an error (which auto-includes stacktrace) :) 
https://github.com/NuGet/NuGet.Services.Metadata/commit/da171b4afbf2a4bbbdc79447ff9a5f6a70e85ba8